ras/lshw.py

```diff
@@ -49,6 +50,10 @@ def run_cmd_out(self, cmd):
 
     def setUp(self):
         sm = SoftwareManager()
+        dist = distro.detect()
+        if dist.name == "SuSE" and dist.version <= 12:
+            self.cancel("lshw is supported on SLES15 or later versions only.")
```

Comment:
can be like 
 ```       if dist.name == "SuSE" and dist.version <= 12:
               self.cancel("lshw  not  supported on SUES-%s " % dist.version ) ```

Reply:
@narasimhan-v @bskjois @kamalesh-babulal above changes will hold good

Reply:
@PraveenPenguin with `self.cancel("lshw  not  supported on SUES-%s " % dist.version )` the user will not informed about the supported version.

Reply:
I vote for the `lshw is supported on SLES15 or later versions only`.

Reply:
@bskjois still on error message i not convinced as I suggested  it should print distro version where it is not able to run (running distro) and skip the test  , as test not  meant for notify user where it is supported

Reply:
@PraveenPenguin I thought even that message will let user know that the test is not supported on any lower versions of SLES15. If not we shall have that message explicitly as "This test is not supported on any version lower to SLES15" OR "lshw not supported on SUES-%s. Please run on SLES15 or higher versions " % dist.version"

This will let the user know that the test is not supported on the current version and also where it is supported as well. is this looks fine?